### PR TITLE
Add a config option to exclude logged in users

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,20 @@ Requires users to accept terms of service before accessing data. Text and langua
...
-The setting applies to shares via link or mail (with and without password protection). Default is off `0`
+The setting applies to shares via link or mail (with and without password protection).
+
+Default is disabled: `0`
 ```
 ./occ config:app:set terms_of_service tos_on_public_shares --value '1'
 ```
+## 👤 Exclude logged in users
+
+To exclude logged in users from the terms of service, set this config to `0`.
```
Comment (wiswedel) [inline README.md]:
```suggestion
To exclude registered users from accepting the terms of service, set this config to `0`.
Therefore, only email- and public link sharees have to accept the terms of service.
```

Comment (nickvergessen) [reply README.md]: Adjusted the headline as well to "registered"

Comment (wiswedel) [reply README.md]: > Adjusted the headline as well to "registered"

👍
makes the setting terminal clear

**Comment by nickvergessen** [comment]
Ignoring 16 for now, until we know if it is needed

**Pull request merged**